src/utils/constants.ts

```diff
@@ -1,3 +1,6 @@
+import dotenv from 'dotenv'
+dotenv.config()
```

Comment:
I think we can move this to the top of the test file(s) so we don't have to have it in here / don't have to run it during production